### ISSUE TITLE
panic: interface conversion: sqlparser.Statement is nil, not *sqlparser.Insert

### ISSUE THREAD

**Issue by vpashovski** [mpchadwick/dbanon#18]
Hello,
I am getting an error when I run dbanon on Magento 2 database. It is simmiar to #14 
```
root@3355fca9767a:/home/dev# mysqldump --complete-insert dev | ./dbanon -config=magento2-mapped.yml > anonymized.sql
panic: interface conversion: sqlparser.Statement is nil, not *sqlparser.Insert

goroutine 1 [running]:
github.com/mpchadwick/dbanon/src.LineProcessor.processInsert(0xaedb67, 0x9, 0xc0002f8e70, 0xbb93c0, 0xc00013a180, 0xc000336920, 0xc0015b4000, 0x35f8b, 0x35f8b, 0x35f8b)
        /Users/maxchadwick/go/src/github.com/mpchadwick/dbanon/src/processor.go:32 +0x1086
github.com/mpchadwick/dbanon/src.LineProcessor.ProcessLine(0xaedb67, 0x9, 0xc0002f8e70, 0xbb93c0, 0xc00013a180, 0xc000336920, 0xc0015b4000, 0x35f8b, 0xc0003ea190, 0x42)
        /Users/maxchadwick/go/src/github.com/mpchadwick/dbanon/src/processor.go:22 +0x9b
main.main()
        /Users/maxchadwick/go/src/github.com/mpchadwick/dbanon/main.go:112 +0x706
mysqldump: Got errno 11 on write
```

**Comment by mpchadwick** [comment]
Would it be possible to share the contents from your mysqldump file that causes this to reproduce? Understand sharing a full dump file could definitely have some privacy concerns but if the specific line(s) where it's choking doesn't have any sensitive data I'd like to take a look.

**Comment by mpchadwick** [comment]
I just published and update (v0.6.2...https://github.com/mpchadwick/dbanon/releases) which will maybe help. Specifically this change

https://github.com/mpchadwick/dbanon/commit/c16d6e1e1cdbcbc0aef956eb35f4292adc19c4eb

It's hard for me to know without having a copy of the the INSERT statement that was being processed.

**Comment by mpchadwick** [comment]
Closing due to inactivity.
